content/tutorials/connect-to-networks.md

```diff
@@ -1,30 +1,22 @@
 ---
-title: Connect to Ocean-Related Networks
-description: On connecting to Ethereum, Rinkeby and Your Local Network.
+title: Connect Wallets to Networks
+description: 
 ---
 
 ## Connect to Ethereum mainnet
 
-Metamask and other ERC20 wallets default to Ethereum mainnet. Therefore your wallet is almost certainly pointing to Ethereum by default.
-
-Some apps may need `network_id` and `chain_id`. The values are both `1` for Ethereum mainnet. `Symbol` and `nickname` can be whatever value you like.
-
-## Connect to Rinkeby testnet
+Ocean's on Ethereum mainnet, for production usage.
 
-In MetaMask, click on the network name dropdown, then select `Rinkeby`.
-
-Its `network_id` is 3 and `chain_id` is 3.
-
-## Connect to local Ganache network
+Metamask and other ERC20 wallets default to Ethereum mainnet. Therefore your wallet is almost certainly pointing to Ethereum by default.
 
-**Metamask.** In MetaMask, click on the network name then click on `Custom RPC` in the drop-down list. Scroll down to the `New Network` section. Enter the above RPC URL. You don't need to add a port number to the end of the RPC URL. Enter the ChainID, Symbol and Nickname if you like. See the [MetaMask docs about how it uses the ChainID](https://metamask.github.io/metamask-docs/Main_Concepts/Sending_Transactions).
+## Connect to public testnets
 
-**Barge.** If you're using Ocean [Barge](https://github.com/oceanprotocol/barge) to help run a local network, you can connect to that local node at RPC URL [http://localhost:8545](http://localhost:8545) (called "Localhost 8545" in MetaMask). You can configure that local node by editing the files in the `barge/networks/pacific/config/` directory.
+`Rinkeby` and `Ropsten` are prominent Ethereum test nets. Ocean is deployed to both.
```

Comment:
stay consistent, either `testnets` like in title, or just write it out here, `test networks`